<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainForm</class>
 <widget class="QWidget" name="MainForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>200</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Format Commit Message</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Module Group</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ModuleGroupEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Member Name</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="MemberNameEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Label Marker</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LabelMarkerEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,1,1,7">
       <item>
        <widget class="QLabel" name="label">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string> EIP Number </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IssueNumberEdit"/>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>EIP Description</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="IssueDescEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="ConvertResultEdit"/>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>LabelMarkerEdit</zorder>
  <zorder>label_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>FormatMessage</zorder>
  <zorder>groupBox_2</zorder>
  <zorder></zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>IssueNumberEdit</zorder>
  <zorder>IssueDescEdit</zorder>
  <zorder>IssueListWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>ModuleGroupEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>MemberNameEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>LabelMarkerEdit</zorder>
  <zorder>ConvertResultEdit</zorder>
  <zorder>ConvertButton</zorder>
  <zorder>horizontalSpacer</zorder>
  <zorder>LabelMarkerEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>MemberNameEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>ModuleGroupEdit</zorder>
  <zorder>label_4</zorder>
  <zorder>LabelMarkerEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>MemberNameEdit</zorder>
  <zorder>label_5</zorder>
  <zorder>ModuleGroupEdit</zorder>
  <zorder>label_4</zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
